caas/kubernetes/provider/constraints.go

```diff
@@ -23,5 +27,80 @@ var unsupportedConstraints = []string{
 func (k *kubernetesClient) ConstraintsValidator(ctx context.ProviderCallContext) (constraints.Validator, error) {
 	validator := constraints.NewValidator()
 	validator.RegisterUnsupported(unsupportedConstraints)
-	return validator, nil
+	return &constraintsValidator{Validator: validator}, nil
+}
+
+type constraintsValidator struct {
+	constraints.Validator
+}
+
+func validateTopologyKeyTagFunc(key string, topologySpreadKey string, tagList []string) error {
```

Comment:
validateTopologyKeyTag

should have a comment even though it's private given it's non trivial behaviour